frontend/src/app/shared/components/cron-builder/cron-builder.component.html

```diff
@@ -1,11 +1,18 @@
 <div class="cron-wr">
   <cron-jobs [(ngModel)]="cronResult"> </cron-jobs>
-  <ng-template [ngIf]="cronResult" [ngIfElse]="elseBlock">
-    <button (click)="saveBuildPeriod()" class="btn btn-info">Add </button>
-   </ng-template>
-
+  <ng-template [ngIf]="cronInput" [ngIfElse]="elseBlock">
+    <div class="btn-wr">
+      <button (click)="updating()" class="btn btn-warning">Update </button>
+      <button (click)="deleting()" class="btn btn-danger">Delete </button>
+    </div>
+  </ng-template>
    <ng-template #elseBlock>
-    <button (click)="saveBuildPeriod()" class="btn btn-secondary" disabled>Add </button>
-   </ng-template>
+    <ng-template [ngIf]="cronResult" [ngIfElse]="NewCron">
```

Comment:
Try to avoid so many ng-templates.
Example:
<div *ngIf="condition; else elseBlock">Content to render when condition is true.</div>
<ng-template #elseBlock>Content to render when condition is false.</ng-template>